src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -6,8 +6,13 @@
   `OpenTelemetry.Exporter` namespace.
   ([#1770](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1770))
 * Changed default port for OTLP Exporter from 55680 to 4317
-* Default ServiceName, if not found in Resource, is obtained from SDK
-  using GetDefaultResource().
+* Default ServiceName, if not found in Resource, is obtained from SDK using
+  GetDefaultResource().
+* Modified the data type of Headers option to string; Added a private method
```

Comment:
Added a private method
  GetMetadataFromHeaders to create a metadata object from the headers passed as
  a string in the options  -- this is optional in changelog.

Reply:
Updated the CHANGELOG.md